content/04. box_1_experimental_design.md

```diff
@@ -0,0 +1,40 @@
+In the testing phase, a new or unseen test dataset or _held-out validation set_ is used to test whether the patterns learned by the model can be generalized to new data points.
+While the evaluation dataset helps the model iteratively update its parameters to learn important patterns in the training data, the held-out validation set helps test the generalizability of the model.
+Generalizability of a model is its ability to recognize patterns that can help predict the class or an outcome for previously unseen data.
+High generalizability of a model on previously unseen data suggest that the model has identified fundamental patterns in the data that may also inform our knowledge regarding the question of interest for which the experiment was designed.
+Generalizability can be affected if _data leakage_ occurs during training of the model, i.e., if a model is exposed to the same or similar data points in both the training set and the held-out test set.
+Ensuring absence of any overlap or relatedness among data points or samples (e.g., samples with familial relationship, samples from same patient) used in the training set and held-out test set is important to avoid _data leakage_ during model training.
+Specifically in cases of rare genetic diseases where many samples can contain familial relationships, special care should be taken while crafting the training and testing sets to ensure that no data leakage occurs and the trained model has high generalizability.
+
+
+##### Training and testing
+
+The implementation of a machine learning experiment begins with splitting a single dataset of interest such that 90% of the data is used for training (generally subdivided into the _training dataset_ and the _evaluation dataset_), and remaining 10% of the data is used for testing or validation (as the _held-out validation dataset_).
+This makes sure that all the datasets involved in training and testing a model maintain uniformity in the features.
+In case of rare diseases where multiple datasets may be combined to make a large enough training dataset, special care is taken to standardize the features and the patterns therein.
+The iterative training stage helps the model learn important patterns in the training dataset and then use the evaluation dataset to test for errors in prediction and update its learning parameters (_hyperparameter tuning_).
+The method by which the evaluation dataset tests the performance of the trained model and helps update the hyperparameters is called _cross-validation_.
+To maximally utilize the available data for cross-validation, there can be multiple approaches to form the training and evaluation datasets e.g. leave-p-out cross-validation, leave-one-out cross-validation, k-fold cross-validation, Monte-Carlo random subsampling cross-validation [@doi:10.1007/978-1-4614-6849-3].
+In case of k-fold cross-validation, a given dataset is shuffled randomly and split into _k_ parts.
```

Comment:
Do we call this out because it's in the figure? or some other reason

Reply:
not quite sure what you are referring to here.. k-fold cross validation?

Reply:
we used k-fold cross validation as an example.. and then added this to the figure for illustrative purposes